src/Microsoft.Data/PrimitiveDataFrameColumnArithmetic.cs

```diff
@@ -0,0 +1,4138 @@
+﻿
```

Comment:
Nit, blank lines

Reply:
I was able to remove 1 of the blank lines in my next patch, but for the life of me, I am not able to find out where the first blank line is coming from. All I know is that it is from some random space in the .ttinclude file